src/app/character-select/dropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import ClickOutside from '../dim-ui/ClickOutside';
+import { DimStore } from '../inventory/store-types';
+import SimpleCharacterTile from '../inventory/SimpleCharacterTile';
+import './dropdown.scss';
+
+interface Props {
+  stores: DimStore[];
+  selectedStore: DimStore;
+  onCharacterChanged(order: string): void;
+}
+
+interface State {
+  dropdownOpen: boolean;
+}
+
+/**
+ * Dropdown for picking a character
+ */
+export default class CharacterDropdown extends React.Component<Props, State> {
```

Comment:
By the way, the convention we've landed on for React components is that the file should match the default exported class - so this would be `CharacterDropdown.tsx`, not `dropdown.tsx`.